src/css/component-styles/OrganizationCard.scss

```diff
@@ -0,0 +1,55 @@
+.organization-card {
+  border-radius: $radius-sm;
+  box-shadow: 0 4px 8px 0 $shadow-black;
+  transition: 0.3s;
+  width: 100%;
+  &:hover {
+    box-shadow: 0 8px 16px 0 $shadow-black;
+  }
+}
+
+.organization-card-header {
+  padding: 15px;
+  background-color: $light-gray;
+  text-align: center;
+  overflow: hidden;
+  border-radius: $radius-sm $radius-sm;
+  display: flex;
+  justify-content: space-between;
+}
+
+.organization-card-header-text {
+  font-size: $font-size-lg;
+  font-weight: bold;
+  letter-spacing: 2px;
+  width: calc(100% - 25px);
+}
+
+.organization-card-body {
+  display: flex;
+  flex-direction: column;
+  padding: $spacing-md;
```

Comment:
missing `line-height: 1.5;` from the original OrganizationCardLayout.js file, May be the reason why the address part is a bit offset.

Reply:
unless `flex` covers that, i am not 100% sure

Reply:
fixed